distiller.nf

```diff
@@ -437,12 +437,12 @@ BWA_INDEX = Channel.from([[
 process map_parse_sort_chunks {
     tag "library:${library} run:${run} chunk:${chunk}"
     storeDir getOutputDir('mapped_parsed_sorted_chunks')
- 
+
     input:
     set val(library), val(run), val(chunk), file(fastq1), file(fastq2) from LIB_RUN_CHUNK_FASTQS
     set val(bwa_index_base), file(bwa_index_files) from BWA_INDEX.first()
     file(chrom_sizes) from CHROM_SIZES_FOR_PARSING.first()
-     
+
     output:
```

Comment:
Overall this looks very good!!!
I have two suggestions. One applies here - if trimming is enabled, fastp will generate an html report. If you do not declare this file in the output section, it will stay in the work folder and users will never find it. However, the difficulty is that this html file won't be generated if trimming is disabled, in which case nextflow won't find it and will fail. The solution is using the optional output statement as described here: https://www.nextflow.io/docs/latest/process.html#optional-output

Reply:
Right now, I have it as: 
file("${library}.${run}.${ASSEMBLY_NAME}.${chunk}.html") optional true 
which stores the html files in 'mapped_parsed_sorted_chunks'. Is that the best place or is processed_fastqs a better location?

Reply:
Perhaps we just leave the fastp reports in the work folders and integrate the fastp.json file into a [multiQC report](https://multiqc.info/docs/#fastp). I think multiqc recursively searches folders (in this case for anything *fastp.json). @golobor, do people normally delete work after the pipeline is finished to save space? Meaning, all valuable information needs to be put into results/

@sergpolly, are you planning on integrating your multiqc work into distiller? And do you have it set up to search the results/ directory or work/?